src/auth/auth.guard.ts

```diff
@@ -17,22 +17,26 @@ export class AuthGuard implements CanActivate {
             request['user'] = payload;
         } 
         catch (e) {
-            console.log(e);
             throw new UnauthorizedException();
         }
         return true;
     }
 
     async extractToken(request: any): Promise<string | undefined> {
-        const cookies: string = request.headers.cookie;
-        if (!cookies) {
-            return undefined;
+        try {
+            const cookies: string = request.headers.cookie;
+            if (!cookies) {
+                return undefined;
+            }
```

Comment:
Just leaving as a simple comment because you can just ignore this, but, I kinda hate that a simple `if <something> return undefined` statement spans 3 lines. 

How about
```suggestion
            if (!cookies) return undefined;
```

Reply:
Hmm, no. I don't like it @HuseyinSimsek7904. Let's go with 

```suggestion
            if (!cookies) 
            {
                return undefined;
            }
```